lib/hanami/commands/new/abstract.rb

```diff
@@ -93,6 +93,10 @@ def git_dir_present?
           File.directory?(target.join('.git'))
         end
 
+        def git_available?
+          system('which', 'git', out: File::NULL)
```

Comment:
@beauby thanks for the PR, it looks good, the only thing I'm concerned that this line doens't work in windows but I can't tell you if hanami works in windows.

cc @jodosha